WriterDesk1/backend/app/ATP/initialiseATP.py

```diff
@@ -157,4 +164,57 @@ def initialiseATP():
     uploadToDatabase(u)
 
     print('participants')
-    print(participants) # keep this print here, and read it 
\ No newline at end of file
+    print(participants) # keep this print here, and read it 
+
+def initialiseATPNoFiles():
+    '''
+        Initialise the database for the ATP in accordance with the specs as given on 20:00, 22-6-2022 
+
+        IMPORTANT: prints 2 participant's username and password
+    '''
+    # clear the database
+    db.session.close()
+    db.drop_all()
+    db.create_all()
+
+    # Delete current files and recreate the upload directory:
+    shutil.rmtree(current_app.config['UPLOAD_FOLDER'])
+    os.mkdir(current_app.config['UPLOAD_FOLDER'])
+    
+    # A user with username ’student@tue.nl’, password ’StudentPass1’ and the student role.
+    uname = 'student@tue.nl'
+    u = User(username=uname, password_plaintext='StudentPass1', role='student')
+    uid = u.id
+    uploadToDatabase(u)
+    assert User.query.filter_by(username=uname).first() is not None
+
+    # A user with username ’researcher@tue.nl’, password ’ResearcherPass1’ and the researcher role.
+    uname='researcher@tue.nl'
+    u = User(username=uname, password_plaintext='ResearcherPass1', role='researcher')
+    uploadToDatabase(u)
+    res = User.query.filter_by(username=uname).first()
+    researcherId = res.id #save id for later
+    assert res is not None
+
+    # A project with projectName ’DeleteMe’ and as userId the id of ’researcher@tue.nl’.
+    p = Projects(userId=researcherId, projectName='DeleteMe')
+    uploadToDatabase(p)
+    pro = Projects.query.filter_by(userId=researcherId).first()
+    projectId = pro.id #save id for later
+    assert pro is not None
+
+    # Two users with usernames ’par ’ and ’par ’, passwords ” and ” and the participant role.
+    # Two ParticipantToProject entries with as userId the ids of ’par ’ and ’par ’ and as projectId the id of ’DeleteMe’.
```

Comment:
Dit moet nog ingevuld worden maar heeft als het goed is niks met jouw deel te maken